apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,111 +18,172 @@ export function InstructorCourseAdminSettings({
-                      ? html`<span class="text-danger">Missing configuration file</span>
-                          ${resLocals.authz_data.has_course_permission_edit &&
-                          !resLocals.course.example_course
-                            ? html`
-                                <form name="add-configuration-form" class="d-inline" method="POST">
-                                  <input
-                                    type="hidden"
-                                    name="__csrf_token"
-                                    value="${resLocals.__csrf_token}"
-                                  />
-                                  <button
-                                    name="__action"
-                                    value="add_configuration"
-                                    class="btn btn-xs btn-secondary mx-2"
-                                  >
-                                    <i class="fa fa-edit"></i>
-                                    <span class="d-none d-sm-inline">Create infoCourse.json</span>
-                                  </button>
-                                </form>
-                              `
-                            : ''} `
-                      : ''}
-                    ${coursePathExists && courseInfoExists
-                      ? resLocals.authz_data.has_course_permission_view
-                        ? html`
-                            <a
-                              href="${resLocals.urlPrefix}/${resLocals.navPage}/file_view/infoCourse.json"
-                            >
-                              infoCourse.json
-                            </a>
-                            ${resLocals.authz_data.has_course_permission_edit &&
-                            !resLocals.course.example_course
-                              ? html`
-                                  <a
-                                    class="btn btn-xs btn-secondary mx-2"
-                                    href="${resLocals.urlPrefix}/${resLocals.navPage}/file_edit/infoCourse.json"
-                                  >
-                                    <i class="fa fa-edit"></i>
-                                    <span>Edit</span>
-                                  </a>
-                                `
-                              : ''}
-                          `
-                        : 'infoCourse.json'
-                      : ''}
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+          <div class="card">
+            <div class="card-header bg-primary text-white d-flex">Course Settings</div>
+            ${!courseInfoExists || !coursePathExists
+              ? html`<div class="card-body pb-0">
+                  ${CourseDirectoryMissingAlert({ resLocals, coursePathExists, courseInfoExists })}
+                </div>`
+              : ''}
+            <form class="card-body">
```

Comment:
The markup here is a little funky. I'd expect to have a single `<div class="card-body">` which may in turn contain other things. Concretely:

```ts
<div class="card-body">
  ${!courseInfoExists || !coursePathExists
    ? ${CourseDirectoryMissingAlert({ resLocals, coursePathExists, courseInfoExists })}
    : ''}
  <form>
    ...
  </form>
</div>
```

Reply:
Updated to a single `<div class="card-body"> in [8e3b7a9](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/8e3b7a92c2e5562595292f74ea7fb09cb6932da6).